MIDIchlorians/MIDIchlorians/ViewController.swift

```diff
@@ -43,6 +43,14 @@ class ViewController: UIViewController {
         assignDelegates()
     }
 
+    override func viewDidAppear(_ animated: Bool) {
```

Comment:
Did not work with viewDidLoad.
I was getting this error:
`Attempt to present TutorialViewController whose view is not in the window hierarchy`

stackoverflow said I should use viewDidAppear so that all the views have been added to window hierarchy. I have no clue what this means but it seems to work (*famous last words*) :)